File: Starters/Starters 51/TVDISC/Solution.c
#include <stdio.h>

int main() {
	int t,i,a,b,c,d;
	scanf("%d",&t);
	for(i=0;i<t;i++){
	    scanf("%d%d%d%d",&a,&b,&c,&d);
	    if(a-c < b-d){
	        printf("First\n");
	    }
	    else if(a-c > b-d){
	        printf("Second\n");
	    }
	    else printf("Any\n");
	}
	return 0;
}

File: Starters/Starters 51/TVDISC/Solution.go
package main

import "fmt"

func main() {
	var t, a, b, c, d int
	fmt.Scan(&t)
	for i := 0; i < t; i++ {
		if a-c < b-d {
			fmt.Println("First")
		} else if a-c > b-d {
			fmt.Println("Second")
		} else {
			fmt.Println("Any")
		}
	}
}
